### PR TITLE
Adding the acquia github key directly into the goagent

### PR DESCRIPTION
**Pull request by Dillon-Dumesnil**
Configuration Pull Request
---

Make sure that the following steps are done before merging:

  - [ ] A DevOps team member has approved the PR.
  - [ ] Are you adding any new default values that need to be overridden when this change goes live? If so:
    - [ ] Update the appropriate internal repo (be sure to update for all our environments)
    - [ ] If you are updating a secure value rather than an internal one, file a DEVOPS ticket with details.
    - [ ] Add an entry to the CHANGELOG.
  - [ ] If you are making a complicated change, have you performed the proper testing specified on the [Ops Ansible Testing Checklist](https://openedx.atlassian.net/wiki/display/EdxOps/Ops+Ansible+Testing+Checklist)?  Adding a new variable does not require the full list (although testing on a sandbox is a great idea to ensure it links with your downstream code changes).

### CODE REVIEW

**Comment by Dillon-Dumesnil** [comment]
FYI @robrap

File: docker/build/go-agent/Dockerfile [hunk 1]
```diff
@@ -70,11 +70,15 @@ RUN \
...
-# setup the github identity
+# !!!!NOTICE!!!! ---- Runner of this pipeline take heed!! You must replace go_github_key.pem and acquia_github_key.pem
+# with the REAL key material that can checkout private github repositories used as pipeline materials. The key material
+# here is faked and is only used to pass CI!
+# setup the go github identity
 ADD docker/build/go-agent/files/go_github_key.pem /var/go/.ssh/id_rsa
+RUN chmod 600 /var/go/.ssh/id_rsa && \
+    chown go:go /var/go/.ssh/id_rsa
+# setup the acquia github identity
+ADD docker/build/go-agent/files/acquia_github_key.pem /var/go/.ssh/id_rsa
```
Comment (macdiesel) [inline docker/build/go-agent/Dockerfile]: This overwrites the github key. You need to give it a different name.

**Review by macdiesel — CHANGES_REQUESTED** [review]

File: docker/build/go-agent/Dockerfile (hunk 1 above)
Comment (macdiesel) [reply docker/build/go-agent/Dockerfile]: And you will also need to modify the marketing jobs that use the key to point to this new location.

**Comment by Dillon-Dumesnil** [comment]
FYI @macdiesel I'm changing this to put the acquia key in the marketing go agent instead of the regular one.

Also, do I need to run these commands after setting up my acquia key as well?

`# setup the known_hosts
RUN touch /var/go/.ssh/known_hosts && \
    chmod 600 /var/go/.ssh/known_hosts && \
    chown go:go /var/go/.ssh/known_hosts && \
    ssh-keyscan -t rsa,dsa github.com > /var/go/.ssh/known_hosts
`

**Review by coryleeio — APPROVED** [review]

**Review by macdiesel — APPROVED** [review]

**Pull request merged**